app-multimedia/yesplaymusic/autobuild/defines

```diff
@@ -0,0 +1,8 @@
+PKGNAME=yesplaymusic
+PKGDES="A beautiful client for Netease Cloud Music"
```

Comment:
An Internet music streaming client.

The home page lists Apple Music playlists, so it's really more than Netease.